healthcheck/healthcheck_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Again with the 2015. I should know this. Let's just make sure it's right before we merge.